src/routes/messages/messages.controller.spec.ts

```diff
@@ -1154,5 +1159,7 @@ describe('Messages controller', () => {
           message: 'Required',
         });
     });
+
+    it.todo('should disable eth_sign');
```

Comment:
This will be included after https://github.com/safe-global/safe-client-gateway/pull/2422 is merged.